dt-backend/src/model/types/user.type.ts

```diff
@@ -17,7 +17,7 @@ export class User {
   mail: string;
 
   @Field((_type) => [AuditLog])
-  audit: AuditLog[];
+  audit?: AuditLog[];
```

Comment:
Is there are a reason why you changed it to optional?
On another note - we don't need the `@Field()` annotations anymore as that's still GraphQL Stuff as can be seen from the import statement at the top!